components/fxa-client/src/internal/oauth/attached_clients.rs

```diff
@@ -19,7 +21,7 @@ impl FirefoxAccount {
         let session_token = self.get_session_token()?;
         let response = self
             .client
-            .attached_clients(&self.state.config, &session_token)?;
+            .get_attached_clients(&self.state.config, &session_token)?;
```

Comment:
By the way, this was not previously exposed over the FFI, but it was so trivial do so that I've gone ahead and exposed it for completeness.